src/main/plugin/iso19139.ca.HNAP/templates/hnaplevel1-english.xml

```diff
@@ -73,31 +88,39 @@
                            </gmd:textGroup>
                         </gmd:PT_FreeText>
                      </gmd:country>
-                     <gmd:electronicMailAddress xsi:type="gmd:PT_FreeText_PropertyType">
-                        <gco:CharacterString>min@dfo-mpo.gc.ca</gco:CharacterString>
+                     <gmd:electronicMailAddress gco:nilReason="missing" xsi:type="gmd:PT_FreeText_PropertyType">
+                        <gco:CharacterString />
                         <gmd:PT_FreeText>
                            <gmd:textGroup>
-                              <gmd:LocalisedCharacterString locale="#fra">min@dfo-mpo.gc.ca</gmd:LocalisedCharacterString>
+                             <gmd:LocalisedCharacterString locale="#fra" />
                            </gmd:textGroup>
                         </gmd:PT_FreeText>
                      </gmd:electronicMailAddress>
                   </gmd:CI_Address>
                </gmd:address>
                <gmd:onlineResource>
                   <gmd:CI_OnlineResource>
-                     <gmd:linkage>
-                        <gmd:URL>https://www.dfo-mpo.gc.ca/</gmd:URL>
+                     <gmd:linkage gco:nilReason="missing">
+                        <gmd:URL />
                      </gmd:linkage>
                      <gmd:protocol>
                         <gco:CharacterString>https</gco:CharacterString>
                      </gmd:protocol>
                   </gmd:CI_OnlineResource>
                </gmd:onlineResource>
+               <gmd:hoursOfService gco:nilReason="missing" xsi:type="gmd:PT_FreeText_PropertyType">
+                  <gco:CharacterString />
+                  <gmd:PT_FreeText>
+                     <gmd:textGroup>
+                        <gmd:LocalisedCharacterString locale="#fra" />
+                     </gmd:textGroup>
+                  </gmd:PT_FreeText>
+               </gmd:hoursOfService>
             </gmd:CI_Contact>
          </gmd:contactInfo>
          <gmd:role>
             <gmd:CI_RoleCode codeList="http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_90"
-                             codeListValue="RI_418">author; auteur</gmd:CI_RoleCode>
+                             codeListValue="RI_414">pointOfContact; contact</gmd:CI_RoleCode>
```

Comment:
Point of contact is a sensible default and author is not?

Reply:
@bo-lu this template was copied from FGP - would you be able to answer @jodygarnett's question?

The only main difference from the FGP template is that we have Department of Fisheries and Ocean in the organization and I'm willing to change it to Canada like FGP template if requested.

Reply:
@jodygarnett  - FGP's suggestion based on their spreadsheet is the "Point of contact".  I would imaging that most people want to know who to contact to get more information.

Reply:
I will merge as the template is better than what it was - if changes are required we can always make another PR